packages/database/src/schema.prisma

```diff
@@ -1205,3 +1205,25 @@ model Rule6Party {
   //Optional link to AppealCase
   AppealCase AppealCase? @relation(fields: [caseReference], references: [caseReference])
 }
+
+/// Proofs of Evidence, Final Comments, Statements, Planning Obligations and Interested Party Comments
+/// Received from BO will share this model and table
+model Representation {
+  /// this ID is for front office only
+  id String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
+
+  representationId        String /// The unique identifier for the representation
+  caseId                  Int? /// Internal case identifier
+  caseReference           String    @unique /// External case identifier
+  status                  String? /// Status of the representation, [ "awaiting_review", "referred", "valid", "invalid", "published", "archived", "draft", "withdrawn", null ]
+  originalRepresentation  String? /// The original representation
+  redacted                Boolean? /// Indicates if the representation is redacted
+  redactedRepresentation  String? /// The redacted version of the representation
```

Comment:
FYI `String` is `nvarchar(1000)` (https://www.prisma.io/docs/orm/reference/prisma-schema-reference#string) - do we need to allow for longer representations here?

Reply:
That's a good point, I think currently we have that limitation in place on the submission side (this table is for the reps once they have been broadcast by BO), so we might need to standardise this across the codebase (including BO)

Reply:
Reverted as per our discussion and wider exercise looking at character limits